consensus/src/round_manager.rs

```diff
@@ -1741,6 +2017,18 @@ impl RoundManager {
                     }
                     break;
                 }
+                opt_proposal = opt_proposal_loopback_rx.select_next_some() => {
```

Comment:
Random order in the select or there is a reason to prioritize opt_proposal_loopback_rx?